common/lib/xmodule/xmodule/word_cloud_module.py

```diff
@@ -91,12 +103,32 @@ class WordCloudFields(object):
         scope=Scope.user_state_summary
     )
 
+    resources_dir = 'assets/word_cloud'
+    template_dir_name = 'word_cloud'
 
-class WordCloudModule(WordCloudFields, XModule):
-    """WordCloud Xmodule"""
-    js = {'js': [resource_string(__name__, 'assets/word_cloud/src/js/word_cloud.js')]}
-    css = {'scss': [resource_string(__name__, 'css/word_cloud/display.scss')]}
-    js_module_name = "WordCloud"
+    preview_view_js = {
+        'js': [
+            resource_string(__name__, 'assets/word_cloud/src/js/word_cloud.js'),
+        ],
+        'xmodule_js': resource_string(__name__, 'js/src/xmodule.js')
```

Comment:
If possible, please add a trailing space for the `xmodule_js`, it seems to be forgotten since you added it elsewhere.